src/common/constants/constants.ts

```diff
@@ -99,7 +99,7 @@ export const localStorageKey = {
 };
 
 export const ApiVersion = {
-    arm: "2016-09-01",
+    arm: "2020-01-01",
```

Comment:
Should this be 2021?

Reply:
`2020-01-01` is the latest ARM API version.
https://docs.microsoft.com/en-us/rest/api/resources/subscriptions/list